Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -238,6 +238,14 @@ public int GetControlSchemeIndex(string name)
             return m_ControlSchemes[index];
         }
 
+        public void RemoveControlScheme(string name)
+        {
+            if (string.IsNullOrEmpty(name))
+                throw new ArgumentNullException("name");
+
+            ArrayHelpers.Erase(ref m_ControlSchemes, GetControlScheme(name));
```

Comment:
Use `GetControlScheme` index and `ArrayHelpers.EraseAt`. The current code first looks up the control scheme and then does an `Equals` comparison of each scheme by going through the default equality comparer. All stuff that can be avoided.

Also, please extend `Actions_CanAddControlSchemeToAsset` to `Actions_CanAddAndRemoveControlSchemeToAsset` test to cover this method.